packages/core/src/sass/app/modules/reality-check.scss

```diff
@@ -141,4 +222,41 @@
             }
         }
     }
+    .dc-input {
+        @include mobile {
+            margin-bottom: 0rem;
+        }
+    }
+    .dc-radio-group {
+        .dc-radio-group__label {
+            @include mobile {
+                font-size: 12px;
+            }
+        }
+        .dc-radio-group__item:not(:last-child) {
+            @include mobile {
+                margin-right: 8px;
+            }
+        }
+        .dc-radio-group__circle {
+            @include mobile {
+                margin-right: 4px;
+            }
+        }
+    }
+    .reality-check__separator {
+        @include mobile {
+            margin: 0rem;
```

Comment:
```suggestion
            margin: 0;
```

Reply:
@zakhov  fixed

Reply:
@shayan-binary-2 can you address this concern?